docs/performance.md

```diff
@@ -496,12 +439,15 @@ const modulePaths = require('./modulePaths');
 const resolve = require('path').resolve;
 const fs = require('fs');
 
+// Update the following line if the root folder of your app is somewhere else.
+const ROOT_FOLDER = path.resolve(__dirname, '..');
+
 const config = {
   getTransformOptions: () => {
     const moduleMap = {};
     modulePaths.forEach(path => {
       if (fs.existsSync(path)) {
-        moduleMap[resolve(path)] = true;
+        moduleMap[resolve(ROOT_FOLDER, path)] = true;
```

Comment:
I think you'll want to resolve the absolute path and pass it into `existsSync` above. Otherwise, LGTM!

Reply:
oh yeah that's right 😆